proposals/collection-expression-arguments.md

```diff
@@ -358,6 +366,9 @@ If the target type is any other type, then a binding error is reported for the *
 ```csharp
 Span<int> a = [with(), 1, 2, 3]; // error: arguments not supported
 Span<int> b = [with([1, 2]), 3]; // error: arguments not supported
+
+int[] a = [with(), 1, 2, 3]; // error: arguments not supported
```

Comment:
A line above says that empty `with()` means the same as no `with`, so why is this an error?

Reply:
(It applies to interfaces, but why only to interfaces?)

Reply:
Ah I remember now, that was LDM choice.

Reply:
Yes.  We explicitly decided there that `with()` was only ok for the destination types where `with(...)` could be ok.  Since there's no `with(...)` ever ok for arrays/spans, we don't allow `with()` either.

Reply:
Is `with()` allowed when "classic collection initializer" pattern is being used? That's what subsection `#### Constructors` is saying?